src/main/java/website/frontrow/board/Player.java

```diff
@@ -19,10 +19,21 @@
      * The points accumulated by the player.
      */
     private int score;
+    
+    /**
+     * The amount of lives the player has.
+     */
+    private int lives;
+    
+    /**
+     * To avoid Checkstyle warnings.
+     * The amount of lives the player starts with is 3.
+     * However, if that integer is final, we can't change it.
+     */
+    private final int three = 3;
```

Comment:
Please change the variable name to DEFAULT_LIVES. It's better code style.

< naming